cmd/loopd/view.go

```diff
@@ -28,37 +30,40 @@ func view(config *config) error {
 	}
 	defer cleanup()
 
-	swaps, err := swapClient.FetchLoopOutSwaps()
-	if err != nil {
+	if err := viewOut(swapClient, chainParams); err != nil {
+		return err
+	}
+
+	if err := viewIn(swapClient, chainParams); err != nil {
 		return err
 	}
 
-	if len(swaps) == 0 {
-		fmt.Printf("No swaps\n")
+	return nil
+}
+
+func viewOut(swapClient *loop.Client, chainParams *chaincfg.Params) error {
```

Comment:
I think we should instead make this an command for the `loop` binary, but it isn't blocking and can be done in a follow up.